map_model/src/pathfind/dijkstra.rs

```diff
@@ -89,9 +102,37 @@ fn calc_path(
     }
     steps.push(PathStep::Lane(req.end.lane()));
     assert_eq!(steps[0], PathStep::Lane(req.start.lane()));
+    // TODO Dijkstra's for vehicles currently ignores uber-turns!
     Some((Path::new(map, steps, req.clone(), Vec::new()), cost))
 }
 
+fn calc_path_v2(
+    graph: DiGraphMap<DirectedRoadID, MovementID>,
+    req: &PathRequest,
+    params: &RoutingParams,
+    map: &Map,
+) -> Option<(Path, Duration)> {
+    let end = map.get_l(req.end.lane()).get_directed_parent(map);
```

Comment:
The implementation is basically the same as the old one, except that the graph is expressed as directed roads, not lanes. Then there's a new `path_v2_to_v1` step at the end to keep the API the same temporarily.